python/doc/pyplots/Dirichlet.py

```diff
@@ -10,16 +10,14 @@
 theta_2 = (0.5, 0.7, 0.9)
 pdf_2d_1 = ot.Graph(title_2d.format(theta_1), "x1", "x2", True)
 pdf_2d_2 = ot.Graph(title_2d.format(theta_2), "x1", "x2", True)
-palette = ot.Drawable.BuildDefaultPalette(10)
 list_theta = [(2, 2), (7, 5), (2, 6), (3, 4)]
 for theta in list_theta:
     distribution = ot.Dirichlet(theta)
     pdf_curve = distribution.drawPDF()
     cdf_curve = distribution.drawCDF()
     pdf_graph.add(pdf_curve)
     cdf_graph.add(cdf_curve)
-pdf_graph.setColors([palette[i] for i in range(4)])
-cdf_graph.setColors([palette[i] for i in range(4)])
+cdf_graph.setColors(pdf_graph.getColors())
```

Comment:
even this line might no longer be necessary if pdf_graph and cdf_graph both use the default palette

Reply:
yes!